docs/Migrating.md

```diff
@@ -11,23 +11,28 @@ The versions can be found in
 ## Migrating from 0.13 to latest
 
 ### Important changes
+* Several changes were made to how agents are reset and marked as done:
+  * Calling `Done()` on the Agent will now reset it immediately and call the `AgentReset` virtual method. (This is to simplify the previous logic in which the Agent had to wait for the next `EnvironmentStep` to reset)
+  * The "Reset on Done" setting in AgentParameters was removed; this is now effectively always true. `AgentOnDone` virtual method on the Agent has been removed.
+  * Agents will always request a decision after being marked as `Done()` and will no longer wait for the next call to `RequestDecision()`.
```

Comment:
Is this true ? I don't think that is right, if it is, it might be a bug.

Reply:
You wrote it :P  I just moved it. I think it was from the first set of changes.

Reply:
That line came from https://github.com/Unity-Technologies/ml-agents/pull/3242